docs/developer-guide/nodejs-api.md

```diff
@@ -220,7 +220,7 @@ The `CLIEngine` is a constructor, and you can create a new instance by passing i
 * `ignore` - False disables use of `.eslintignore`, `ignorePath` and `ignorePattern` (default: true). Corresponds to `--no-ignore`.
 * `ignorePath` - The ignore file to use instead of `.eslintignore` (default: null). Corresponds to `--ignore-path`.
 * `ignorePattern` - Glob patterns for paths to ignore. String or array of strings.
-* `parser` - Specify the parser to be used (default: `espree`). Corresponds to `--parser`.
+* `parser` - Specify the parser to be used (default: `espree`). Corresponds to `--parser`. You can also pass an object with the `parser` or `parseForESLint` function.
```

Comment:
Apologies, my previous comment was a little subtle. In addition to asking you to add a reference to `parseForESLint`, I also wanted you to change the last `parser` to `parse`. Would you mind making this change as well? Thanks!